test/performance/commCompare/gbt/many-to-one.chpl

```diff
@@ -0,0 +1,98 @@
+use Time;
+use CommDiagnostics;
+
+const numWorkerNodes = numLocales - 1;
+config const numTasksPerNode = min reduce ([i in 1..numWorkerNodes]
+                                           Locales(i).maxTaskPar);
+
+config const numOps = 10**6;
+const numOpsPerTask = numOps / (numWorkerNodes * numTasksPerNode);
+
+config const printConfig = false;
+config const printTimings = false;
+config const printCommDiags = false;
+
+var x0: int;
+var x0Atomic: atomic int;
+
+enum op_t {
+  opNone,
+  opGet,
+  opPut,
+  opAMO,
+  opFastOn,
+  opOn
+};
+
+use op_t;
+
+config param op = opNone;
+
+proc main() {
+  if printConfig {
+    writeln(numWorkerNodes, ' worker nodes, ',
+            numTasksPerNode, ' worker tasks per node, ',
+            numOps, ' ', op, (if numOps == 1 then '' else 's'));
+  }
+
+  if printCommDiags {
+    resetCommDiagnostics();
+    startCommDiagnostics();
+  }
+
+  const startTime = getCurrentTime();
+
+  coforall locIdx in 1..numWorkerNodes with ( ref x0, ref x0Atomic ) {
```

Comment:
I think the default intent for atomics is already ref, can you drop the `ref x0Atomic`?

Reply:
Will do.